pagaya_interview_ofir.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<p>should be better: <code>dfgdsfsdgfsdgfsg</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DoronGurel1/NB_review_test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>